documentation/operators/src/faq/smoosh-faq.md

```diff
@@ -0,0 +1,88 @@
+
+### Can I run a mix node only?
+
+Yes, to run a mix node only is an option. However it will be less rewarded as nodes providing option for gateway due to the work and risk their operators have.
+
+## Token Economics & Rewards
+
+### What are the incentives for the node operator?
+
+In the original setup there were no incentives to run a `network-requester`. After the transition all the users will buy multiple tickets of zkNyms credentials and use those as [anonymous e-cash](https://arxiv.org/abs/2303.08221) to pay for their data traffic (Nym API will do the do cryptographical checks to prevent double-spending). All collected fees get distributed to all active nodes proportionally to their work by the end of each epoch.
+
+### How does this change the token economics?
+
+The token economics will stay the same as they are, same goes for the reward algorithm. In practice the distribution of rewards will benefit more the operators who run open gateways.
+
+### How are the rewards distributed?
+
+As each operator can choose what roles their nodes provide, the nodes which work as open gateways will have higher rewards because they are the most important to keep up and stable. Besides that the operators of gateways may be exposed to more complication and possible legal risks.
+ 
+The nodes which are initialized to run as mix nodes and gateways will be chosen to be on top of the active set before the ones working only as a mix node. 
+
+We are considering to turn off the rewards for non-open gateways to incentivize operators to run the open ones. Mix nodes on 'standby' will not be rewarded (as they are not being used). 
+
+The more roles an operator will allow their node to provide the bigger reward ratio which will have huge performance benefits for the end-users.
+
+
+### How will be the staking and inflation after project Smoosh?
+
+We must run tests to see how many users pay. We may need to keep inflation on if not enough people pay to keep high quality gateways on in the early stage of the transition. That would mean keeping staking on for gateways. Staking will always be on for mix nodes.
+
+### When project smooth will be launched, it would be the mixmining pool that will pay for the gateway rewards based on amount of traffic routed ?
+
+Yes, the same pool. Nym's aim is to do minimal modifications. The only real modification on the smart contract side will be to get into top X of 'active set' operators will need to have open gateway function enabled.
+
+### What does this mean for the current delegators?
+
+From an operator standpoint, it shall just be a standard Nym upgrade, a new option to run the gateway software on your node. Delegators should not have to re-delegate.
```

Comment:
`From an operator standpoint, it shall just be a standard Nym upgrade` doesn't seem right - surely people will have to change any scripts / automation, since the node will be running with different flags?

Reply:
Wondering how to wrap this in a better sentence, these were Harry's words.

Reply:
Check in with Andrew about this maybe